docs/resources/faq.md

```diff
@@ -0,0 +1,187 @@
+
+### Do I have to be "tech-smart" to build Trio?
+No. You do not need any experience in code or computers to build Trio. If you own a computer or tablet and an iPhone, you already have the required experience. Beyond that, read the directions slowly and diligently. All the information you will need is in these documents.
+
+Often, the non-tech people do better than the tech people in building Trio. Why? Because the non-tech people take the time to read slowly and look at the screenshots in the directions. The tech people often skim and miss steps, which then leads to build errors that have to be retraced and fixed.
+
+### How long does it take to build Trio?
+The answer to this question is very user-dependent. If you are new to building and need to immerse yourself in the documentation, then a few hours. However, some users like to break this down into smaller, more manageable chunks and complete the task over 24 hours. If you are a repeat builder familiar with the documentation, build, and update process, then 10 - 15 minutes. 
+There are two options for building at present:
+* The 'Browser Build' method using GitHub.
+* The 'Mac Build' method using XCode.
+
+### Do I need to own an Apple computer?
+No, you do not need to own an Apple computer. The 'Browser Build' method can be completed on any computer with access to the Internet and GitHub. However, if you chose the 'Mac Build' method, while you do not necessarily need to own an Apple computer, you must at least borrow one. 
+
+### How often will I need to access the computer to maintain Trio?  
+When you use the 'Browser Build' method, you must access a browser at least once every 90 days to update with the Browser. This is simple enough to do the steps on your phone in just a few minutes. 
+
+When you use Build with Mac: 1. When you first build. 2. Once per year, at the minimum, after that. However, if you decide to use a free Apple Developer account, you must rebuild every 7 days. 
+
+!!! important "Important" 
+_Periodically, developers will release updates to Trio. These updates may include new features or important safety updates. It is recommended that as updates are released, users update their Trio app._
+
+### Do I have to rebuild Trio if I change insulin pumps or CGMs?
+No. Trio lets you move between different pumps and CGM configurations within the app. 
+
+### I am moving from another closed-loop system. Will my settings transfer over?
+If you've been using other closed-loop solutions, it's essential to understand the differences between the one you're used to and Trio. Before making the transition, we recommend that you take the time to read through the documentation to build a basic understanding of Trio. Detailed information on transitioning from other closed-loop systems can be found [here](.../Configuration/transition-qa.md)
```

Comment:
There is an extra dot in the URL which should be:

```markdown
[here](../Configuration/transition-qa.md)
```

Reply:
Corrected